industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,17 @@
+# GSK Industry Benchmark
+Prepared by Alexander Williams
+## Renin MCS Set
+## Software Used
+1. Maestro Protein Prep v.2024-02
+## Protein Preparation: Steps Taken
+1. Protein from original structures was imported into Maestro and using the protein preparation panel the interactive mode was activated following settings were applied on the preprocess panel.
+### Maestro Protein Preparation Workflow
+   1. Cap Termini
+   2. Convert selenomethionines to methionines
+   3. Include Peptides when capping termini.
+2. Final NMA peptide prepared by maestro was converted to NME by using the builder panel and selecting other edits ... change atom properties.
+3. Cofactor NAG was extracted from structure and saved to cofactor.sdf
```

Comment:
This is a glycosylation site - i.e. bound to the nearby ASN. Given it's far from the binding site and we won't be able to handle the glycosylation properly, we probably just should remove it and revert to a normal ASN.

Reply:
Thank you for removing the NAG cofactor! Could you update the preparation details to reflect those changes? Thank you!

Reply:
Updated preparation details.